playbook.yml

```diff
@@ -0,0 +1,68 @@
+- name: Install To-Do on new web servers
+  hosts: webservers
+  become: yes
+  remote_user: ec2-user
+
+  vars_prompt:
+
+    - name: trellokey
+      prompt: What is your trello api key?
+      
+
+    - name: trellotoken
+      prompt: What is your trello api token?
+    
+    - name: trelloboardid
+      prompt: What is your trello board id?
+      
+
+    - name: trellotodoid
+      prompt: What is your trello todo list id?
+
+    - name: trellodoneid
+      prompt: What is your trello done list id?
+
+  tasks:
+  - name: Install git
+    yum:
+      name: git
+      state: latest
+
+  - name: Install python 3
+    yum:
+      name: python3
+      state: present
+  
+  - name: intall poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+
+  - name: Create folder for the ToDo
+    ansible.builtin.file:
+      path: ~/opt/todoapp
+      state: directory
+    become: yes
+
+  - name: Git checkout
+    ansible.builtin.git:
+      repo: 'https://github.com/jknights2000/DevOps-Course-Starter.git'
+      dest: /opt/todoapp
+  
+  - name: intall poetry depend
+    ansible.builtin.shell:
+      cmd: ~/.local/bin/poetry install
+      chdir: /opt/todoapp
+
+  - name: make env
+    ansible.builtin.template:
+      src: ~/env.j2
+      dest: ~/opt/todoapp/.env
+      owner: ec2-user
+      group: wheel
+      mode: '0644'
+
+  - name: Start todoapp
+    ansible.builtin.systemd:
+      state: started
+      name: todoapp.service
```

Comment:
Did you manually create a todoapp.service file in the correct place on the host machine? I don't see anything.

We don't want even a single step that has to be carried out manually on the host machine.

The aim is that if you needed to set up a brand new server, all you need to do is set up the ssh key, update the inventory file and run the playbook.

If it works correctly, you should be able to load up your todo app by visiting http://3.10.250.95:5000/ in your browser (and I should be able to as well).

Please add the todoapp.service file to your git repository, and add an Ansible task that copies it to the correct place (the correct location is given in the PDF).